### ISSUE TITLE
how can i change a value in a view?

### ISSUE THREAD

**Issue by DanielCollins** [jaspervdj/digestive-functors#121]
after using `postForm`, if there are form errors, i get a `view` value. I cannot work out how to reset some of the form values held in the view before re-displaying the form. Specifically, my captcha should be regenerated with each request so as to prevent unlimited captcha attempts.

**Comment by winterland1989** [comment]
I came across the exactly problem as yours, here 's my solution:
```haskell
import           Text.Digestive.Types
import           Text.Digestive.View
...
lform <- getForm "profile" (profileForm u')
let lform' = lform {viewInput = [
    (toPath "email", TextInput $ sUserEmail user)
    ,   (toPath "desc", TextInput $ sUserDesc user)
    ]}
...
```

Not nice but do solve the problem,  we should providing a helper to do this kind of things, for example, set some pre-defined values in views.

**Comment by DanielCollins** [comment]
thanks @winterland1989 this is workable

**Comment by winterland1989** [comment]
@jaspervdj  Do you think we should add some APIs to poke around view?

**Comment by jaspervdj** [comment]
Yes, I wouldn't mind having an API which allows you to set these values in a more convenient way.
